### DISCUSSION TITLE
Any BIOS direction/suggestion?

### DISCUSSION THREAD

**Discussion by EduGuert** [rg35xx-cfw/rg35xx-cfw.github.io#163] [General]
Does anyone have a good BIOS library for the BATOCERA of the RG35XX H?
I'm new to this world of emulators and I don't know where to get this kind of thing.
Sorry if here it's not the right place to ask these kind of things.

**Comment by acmeplus** [comment]
Sorry, but this is not the place to ask. You are supposed to have your own copies of the bios, e.g. amiga ones can be purchased from amigaforever.
